onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -1237,6 +1293,22 @@ public void onClick(View arg0) {
         }
     }
 
+    // Check user profile restrictions on social features
+    private void checkSocialEnabled() {
+        if (Build.VERSION.SDK_INT > 18) {
```

Comment:
Instead of hard-coding `18` here, please use `Build.VERSION_CODES.JELLY_BEAN_MR2` instead.